tests/dummy/app/templates/getting-started/assets.hbs

```diff
@@ -79,69 +79,66 @@ module.exports = function(defaults) {
-    <tr><th>tables</th><td>@import "ember-bootstrap/tables.less";</td><td>@import "ember-bootstrap/bootstrap/tables.scss";</td><td>@import "ember-bootstrap/tables.scss";</td></tr>
-    <tr><th>theme</th><td>@import "ember-bootstrap/theme.less";</td><td>@import "ember-bootstrap/bootstrap/theme.scss";</td><td></td></tr>
-    <tr><th>thumbnails</th><td>@import "ember-bootstrap/thumbnails.less";</td><td>@import "ember-bootstrap/bootstrap/thumbnails.scss";</td><td></td></tr>
-    <tr><th>tooltip</th><td>@import "ember-bootstrap/tooltip.less";</td><td>@import "ember-bootstrap/bootstrap/tooltip.scss";</td><td>@import "ember-bootstrap/tooltip.scss";</td></tr>
-    <tr><th>transitions </th><td></td><td></td><td>@import "ember-bootstrap/transitions.scss";</td></tr>
-    <tr><th>type</th><td>@import "ember-bootstrap/type.less";</td><td>@import "ember-bootstrap/bootstrap/type.scss";</td><td>@import "ember-bootstrap/type.scss";</td></tr>
-    <tr><th>utilities</th><td>@import "ember-bootstrap/utilities.less";</td><td>@import "ember-bootstrap/bootstrap/utilities.scss";</td><td>@import "ember-bootstrap/utilities.scss";</td></tr>
-    <tr><th>variables</th><td>@import "ember-bootstrap/variables.less";</td><td>@import "ember-bootstrap/bootstrap/variables.scss";</td><td>@import "ember-bootstrap/variables.scss";</td></tr>
-    <tr><th>wells</th><td>@import "ember-bootstrap/wells.less";</td><td>@import "ember-bootstrap/bootstrap/wells.scss";</td><td></td></tr>
-  </tbody>
-</table>
+<p>
+  It is possible to import only the assets your app needs but the definitions varies according to the chosen configuration.
+</p>
+
+<p><b>BS3 - Less</b></p>
+<pre><code class="language-less">// app.less
+
+// Required
+@import "ember-bootstrap/variables.less";
+@import "ember-bootstrap/mixins.less";
+
+// Optional - Bootstrapping/Resetting
+@import "ember-bootstrap/bootstrap/normalize.less";
+@import "ember-bootstrap/bootstrap/print.less";
+
+// Optional - Everything else
+@import "ember-bootstrap/scaffolding.less";
+@import "ember-bootstrap/type.less";
+@import "ember-bootstrap/buttons.less";
+</code></pre>
+
+<p><b>BS3 - Sass</b></p>
+<pre><code class="language-sass">// app.scss
+
+// Required
+@import "ember-bootstrap/bootstrap/variables.scss";
+@import "ember-bootstrap/bootstrap/mixins.scss";
+
+// Optional - Bootstrapping/Resetting
+@import "ember-bootstrap/bootstrap/normalize.scss";
+@import "ember-bootstrap/bootstrap/print.scss";
+
+// Optional - Everything else
+@import "ember-bootstrap/bootstrap/scaffolding.scss";
+@import "ember-bootstrap/bootstrap/type.scss";
+@import "ember-bootstrap/bootstrap/buttons.scss";
+</code></pre>
+
+<p><b>BS4 - Sass only</b></p>
+<pre><code class="language-sass">// app.scss
+
+// Required
+@import "ember-bootstrap/variables.scss";
+@import "ember-bootstrap/mixins.scss";
+@import "ember-bootstrap/functions.scss";
```

Comment:
IIRC,`functions` has to be the first import, as `mixins` relies on it! At least that's what they do: https://github.com/twbs/bootstrap/blob/v4-dev/scss/bootstrap.scss#L8-L10

Reply:
Done